### PR TITLE
just: build offline documentation

### PR DESCRIPTION
**Pull request by KAction**

### CODE REVIEW

**Comment by KAction** [comment]
@xrelkd @jk @adamcstephens I still can't figure when github automatically ping maintainers and when it does not, so I'll do it manually.

**Comment by Mic92** [comment]
Result of `nixpkgs-review pr 229591` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li>dogdns</li>
    <li>dogdns.man</li>
  </ul>
</details>
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>just</li>
    <li>just.doc</li>
    <li>just.man</li>
    <li>kabeljau</li>
  </ul>
</details>

**Comment by Mic92** [comment]
dogdns build failure looks unrelated.

**Comment by adamcstephens** [comment]
@Mic92 did you mean to close this?

**Comment by Mic92** [comment]
Wrong button

File: pkgs/development/tools/just/default.nix [hunk 1]
```diff
@@ -42,13 +44,24 @@ rustPlatform.buildRustPackage rec {
     cp $TMPDIR/string.rs tests/string.rs
   '';
 
+  postBuild = ''
+    cargo run --package generate-book
+
+    # No linkcheck in sandbox
+    echo 'optional = true' >> book/en/book.toml
+    mdbook build book/en
+    find .
```
Comment (06kellyjac) [inline pkgs/development/tools/just/default.nix:53]: this `find` can go next update i guess

**Pull request merged**